src/createRoom.js

```diff
@@ -174,6 +174,9 @@ export function findDMForUser(client, userId) {
             return member && (member.membership === "invite" || member.membership === "join");
         }
         return false;
+    }).sort((r1, r2) => {
+        return r2.getLastActiveTimestamp() -
+            r1.getLastActiveTimestamp();
     });
```

Comment:
We can inline this whole thing:
```suggestion
    }).sort((r1, r2) => r2.getLastActiveTimestamp() - r1.getLastActiveTimestamp());
```

Reply:
might be too long for lint? will just merge